### DISCUSSION TITLE
Adding themed classes for borders

### DISCUSSION THREAD

**Discussion by Splinteer** [saadeghi/daisyui#2972] [Ideas / feature requests]
If I want to add a border to a specific element, I'll not be able to set the color using my theme like this:
border-primary
border-accent

To do so I need to use the tailwind classes. 
It applies to every styling related to colors.

**Comment by saadeghi** [comment]
If it's for a specific element, you can use a prefix like `[[data-theme=cupcake]_&]:` for any utility class and it will only be applied to `cupcake` theme.  
Example: https://play.tailwindcss.com/UEp1k5ZPUE

Let me know if you have a question.
